tutorials/Consistent_Scores.ipynb

```diff
@@ -0,0 +1,465 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Generate a list of thresholds of interest\n",
+    "thetas = murphy_thetas([fcst1, fcst2], obs, \"quantile\")\n",
+    "\n",
+    "ms1 = murphy_score(fcst1, obs, thetas, functional=\"quantile\", alpha=0.9)\n",
+    "ms2 = murphy_score(fcst2, obs, thetas, functional=\"quantile\", alpha=0.9)\n",
+    "\n",
+    "# Rename date variable for plotting\n",
+    "ms1 = ms1.rename({\"total\": \"Mean elementary score\", \"theta\": \"wind speed decision threshold (kts)\"})\n",
+    "ms2 = ms2.rename({\"total\": \"Mean elementary score\", \"theta\": \"wind speed decision threshold (kts)\"})\n",
+    "\n",
+    "# Plot the results\n",
+    "ms1[\"Mean elementary score\"].plot()\n",
+    "ms2[\"Mean elementary score\"].plot()\n",
+    "plt.title(\"fcst1 (blue), fcst2 (orange)\")\n",
+    "plt.suptitle(\"Murphy Score (for 90th percentile)\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "What we can see is that fcst2 (orange) performs best for most decision thresholds. Despite weighting higher wind speed thresholds more heavily and fcst2 performing worse for thresholds above 50kt, fcst2 still scored better because it performed substantially better for many decision thresholds and the weights did not exponentially increase."
```

Comment:
This diagram threw me for a while, because visually fcst1 looks pretty much identical to the first diagram. On close inspection, it doesn't even reach 0.08 whereas in the first diagram it gets close to 0.1. It would probably be worth setting the y-axis range to be the same. It may or may not be worth reproducing the first plot side-by-side with this plot so they can be directly compared. Also (and this is probably because I'm not totally familiar with this score and it's a bit late), what's the relationship between the wind speed decision threshold and the 90th percentile? Is this a diagram of the score for a bunch of different consistent quantile scores, or is this saying that tuning for the 90th percentile also just happens to result in something that is better for most decision thresholds, i.e. that almost everyone is better off at ever decision threshold if the service definition is based on the 90th percentile rather than the median?

Reply:
I don't think it really makes sense to compare the two diagrams as the two examples are separate. I have made the y-axes range the same though.

> what's the relationship between the wind speed decision threshold and the 90th percentile? Is this a diagram of the score for a bunch of different consistent quantile scores, or is this saying that tuning for the 90th percentile also just happens to result in something that is better for most decision thresholds, i.e. that almost everyone is better off at ever decision threshold if the service definition is based on the 90th percentile rather than the median?

The wind speed decision threshold, is the threshold that a user makes a decision on. E.g., "I won't go sailing today if the 90th percentile wind forecast is greater than 25 knots". 25 knots is the decision threshold in that case. A value on a Murphy Diagram shows the mean economic regret of a user making decisions at that threshold. The area under the first Murphy Diagram is proportional to the MSE, the area under the second Murphy Diagram is proportional to the mean quantile score. You can imagine the $g'$ and $\phi''$ functions as weighting the area under the Murphy Diagram across the relevant importance of different user decision thresholds. See Fig 7b in https://rmets.onlinelibrary.wiley.com/doi/full/10.1002/qj.4206 for a simple graphical illustration.

Reply:
Resolving this thread following a discussion on how to make some tweaks to the notebook to help aid the reader